x/expiration/client/cli/tx.go

```diff
@@ -0,0 +1,138 @@
+package cli
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/version"
+
+	"github.com/provenance-io/provenance/x/expiration/types"
+)
+
+const (
+	FlagSigners = "signers"
+)
+
+// NewTxCmd is the top-level command for expiration CLI transactions
+func NewTxCmd() *cobra.Command {
+	txCmd := &cobra.Command{
+		Use:                        types.ModuleName,
+		Aliases:                    []string{"exp"},
+		Short:                      "Transaction commands for the expiration module",
+		DisableFlagParsing:         true,
+		SuggestionsMinimumDistance: 2,
+		RunE:                       client.ValidateCmd,
+	}
+	txCmd.AddCommand(
+		ExtendExpirationCmd(),
+		InvokeExpirationCmd(),
+	)
+	return txCmd
+}
+
+// ExtendExpirationCmd creates a command for extending an expiration
+func ExtendExpirationCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "extend [module-asset-id] [duration n{y|w|d|h}]",
+		Aliases: []string{"e"},
+		Short:   "Extend expiration metadata for an asset on the provenance blockchain",
+		Long: `Extend expiration metadata for an asset on the provenance blockchain
+
+Example:
+$ %s tx expiration extend pb1skjwj5whet0lpe65qaq4rpq03hjxlwd9nf39lk duration 1y
+
+module-asset-id   - module asset address
+duration          - the duration period for which the module asset will continue to remain on chain.
+					Valid time units are "y", "w", "d", "h"`,
```

Comment:
This line is probably spaced in too much. Also, if you want it to line up nicely with the beginning of the line above it, you'll need to use spaces (not tabs).